### DISCUSSION TITLE
Firebase labels my Confirmation Emails with Cloud Functions as " Spammy Corp.". How can I change it to a different label?

### DISCUSSION THREAD

**Discussion by Verson-tech** [firebase/functions-samples#934] [Q&A] [answered]
I have followed step by step guide [functions-samples/email-confirmation/](https://github.com/firebase/functions-samples/tree/main/email-confirmation) to create email-confirmation for my page
Firebase, however, labels my confirmation letters with  " Spammy Corp.". 
How can I change it to a different label?

**Comment by jhuleatt** [comment, marked as answer]
Hi @Verson-tech, you need to edit this line to change the `from` field:

https://github.com/firebase/functions-samples/blob/cb13b84f597a5d7e9b847d7d2bf8b8f7b56e0958/email-confirmation/functions/index.js#L43

**Reply by Verson-tech** [reply]
Thank you @jhuleatt.
